docs/clean/user_guide/executing_a_clean.rst

```diff
@@ -0,0 +1,62 @@
+.. _clean_doc_usage_and_args:
+Executing a Clean
+=================
+
+This page contains the basic arguments required to use pyATS Clean. There are two different methods of using pyATS Clean
```

Comment:
weird sentence This page contains the basic arguments required to use pyATS Clean. 

This page explains how to execute pyATS Clean and its arguments. ?